lua/astrocommunity/pack/clojure/clojure.lua

```diff
@@ -0,0 +1,54 @@
+-- Clojure support with Conjure plugin
+local utils = require "astronvim.utils"
+return {
+  -- Clojure Language Server
+  {
+    "williamboman/mason-lspconfig.nvim",
+    opts = function(_, opts) opts.ensure_installed = utils.list_insert_unique(opts.ensure_installed, "clojure_lsp") end,
+  },
+  -- Clojure parser
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = function(_, opts)
+      if opts.ensure_installed ~= "all" then
+        opts.ensure_installed = utils.list_insert_unique(opts.ensure_installed, "clojure")
+      end
+    end,
+  },
+  -- Parinfer parens management for Clojure
+  {
+    "gpanders/nvim-parinfer",
+    ft = { "clojure" },
+    init = function()
+      vim.g.parinfer_force_balance = true
+      vim.g.parinfer_comment_chars = ";;"
+    end,
+  },
+  -- Conjure plugin for Clojure REPL
+  {
+    "Olical/conjure",
+    -- load plugin on filetypes
+    ft = { "clojure" },
+    init = function()
+      vim.g["conjure#mapping#prefix"] = ","
```

Comment:
```suggestion
```

I think actually we should keep this as `localleader` I think the main thing is AstroNvim should provide a default `localleader`. This will make it easier for users to customize things still on their end. I don't mind just merging this in in the meantime but I think the next release of AstroNvim will just have `localleader` defined as probably `,`

Reply:
Yes, I would prefer a localleader set in AstroNvim and/or via the AstroNvim user_example 

Before creating the Clojure pack I used the `options.lua` configuration file from the AstroNvim user_example template to set the localleader, using `g = { maplocalleader = "," }`

For the moment, I'll remove the conjure mapping prefix line and add a note to the readme stating that localleader should be set in the user config, if not already set in AstroNvim.